addons/mail/wizard/mail_compose_message.py

```diff
@@ -193,6 +193,12 @@ def send_mail_action(self):
         # TDE/ ???
         return self.send_mail()
 
+    @api.multi
+    def send_mail_and_schedule_next_action(self):
```

Comment:
Now I am ok with it. While we are at it could we have something like
 - send_mail: basic method sending mails
 - action_send_mail (old send_mail_action): return the act_close + infos mail_sent
 - action_send_mail_and_schedule: calls actino_send_mail + update infos ?

That way we have clear methods with their own output. What do you think about that ?

Reply:
Totally agree

Reply:
Actually almost totally agree, in action_send_mail_and_schedule we can return the complete act_close with schedule_next and call send_mail not action_send_mail

    @api.multi
        def action_send_mail(self):
            self.send_mail()
            return {'type': 'ir.actions.act_window_close', 'infos': 'mail_sent'}

    @api.multi
        def action_send_mail_and_schedule(self):
            self.send_mail()
            return {'type': 'ir.actions.act_window_close', 'infos': 'schedule_next'}

Reply:
Yep, even better.